docs/guides/nrfconnect_examples_software_update.md

```diff
@@ -118,7 +118,72 @@ Complete the following steps to perform DFU using mcumgr:
             hash: cbd58fc3821e749d3abfb00b3069f98c078824735f1b2a333e8a1579971e7de1
         Split status: N/A (0)
 
-7.  Reset the device with the following command to let the bootloader swap
+> **_NOTE:_** In case you use nRF5340DK board that supports multi-image device firmare upgrade,
+> complete 7-9 steps. In other case you should omit them and go to the step 10.
+
+7.  Upload the network core firmware image to the device by running the following command in
+    your example directory:
+
+        $ sudo mcumgr --conntype ble --hci ble-hci-number --connstring peer_name='ble-device-name' image upload build/zephyr/net_core_app_update.bin -n 1
+
+    The operation can take few minutes. Wait until the progress bar reaches
```

Comment:
```suggestion
    The operation can take a few minutes. Wait until the progress bar reaches
```
"Few minutes" = it doesn't take long at all.
"A few minutes" = it can take a while.